tests/assemblers/test_lightgbm.py

```diff
@@ -110,3 +111,73 @@ def test_regression():
             ast.BinNumOpType.ADD))
 
     assert utils.cmp_exprs(actual, expected)
+
+
+def test_regression_from_booster():
+    estimator = lightgbm.LGBMRegressor(n_estimators=2, random_state=1, max_depth=1)
+    utils.train_model_regression(estimator)
+
+    with utils.tmp_dir() as tmp_dirpath:
+        filename = os.path.join(tmp_dirpath, "model.txt")
+        estimator.booster_.save_model(filename)
+        booster = lightgbm.Booster(model_file=filename)
```

Comment:
nit: there's a `model_str` argument which potentially allows not to use a temporary file (in conjunction with `Booster.dump_model` method). I might be wrong however as I didn't verify it myself.